src/Mentat.UI/Models/Assignment.cs

```diff
@@ -1,23 +1,106 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using System.IO;
 using System.Linq;
 using System.Web;
+using Mentat.UI.Controllers;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.EntityFrameworkCore;
 
 namespace Mentat.UI.Models
 {
+    public class FileExtensionCheckAttribute : ValidationAttribute
+    {
+        public FileExtensionCheckAttribute()
+        {
+            Assignment assignment = new Assignment();
+            if (assignment.AssignmentType != null)
```

Comment:
You declared assignment on line 18. Why are you if elsing assignment.AssignmentType? It either is or isn't. Did you mean to do something different on line 18?

Reply:
This will not affect the program at runtime. I'm supposed to pass in "assignment" but I had much trouble doing so. This part is only for a specific error message. It should be updated once I get help.

Reply:
Got ya, thanks for clarification